source/Cosmos.System2_Plugs/System/IO/CosmosFileSystem.cs

```diff
@@ -27,10 +27,10 @@ public static class CosmosFileSystem
          */
         private static string mCurrentDirectory = string.Empty;
 
-        public static void CreateDirectory(object aThis, string fullPath)
+        public static void CreateDirectory(string fullPath)
```

Comment:
Removing `object aThis` does not break the plug here?

Reply:
The file systems are no longer objects, it's now a static class.